transformer_engine/common/normalization/rmsnorm/rmsnorm_bwd_kernels.cuh

```diff
@@ -111,6 +118,16 @@ __global__ __launch_bounds__(Ktraits::THREADS_PER_CTA) void rmsnorm_bwd_tuned_ke
       }
     }
 
+    maybe_t<Ovec[LDGS], FusedAdd> add;
```

Comment:
Future optimization: Load `add` directly into `dx` to avoid extra register usage. However, this would require that `Ivec` and `Ovec` are identical types.